### PR TITLE
update general readme

### PR DESCRIPTION
**Pull request by Tinkaa**
Updating the readme. 
Not sure if the repo structure is an overkill

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,78 @@
...
+financing, activities, and a trigger. 
+The trigger defines an exceptional shock (cause and impact) to which the framework should respond. 
+The framework activates when projections indicate that the shock is likely to realise. 
+Funds are then rapidly disbursed and interventions are implemented. 
+
+The trigger is based on data and enables the automated activation of the framework. 
+This repository contains the triggers that are part of the several AAF pilots. 
+While the triggers are adjusted to the local context, most of the data processing is transferable to other countries.
+
+The AAF pilots implemented as of January 2021 are:
```
Comment (hannahker) [inline README.md:14]: Are there any completed trigger cards that we could link to (the templates that Josee created)?

**Review by hannahker — APPROVED** [review]
Looks great to me :) And good idea to update this now that it's public,

File: README.md (hunk 1 above)
Comment (Tinkaa) [reply README.md:14]: @joseepoirier any completed trigger cards we could link to? 😄 Or other public documents (for Bangladesh maybe the blog?)?

Comment (Tinkaa) [reply README.md:14]: Now included link to Bangladesh blogpost. @joseepoirier let us know if there is any other public material that would be good to link to :) Merging for now.

**Pull request merged**